source/default/tokens/sass/non-printing/functions/_map-getters.scss

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
This is new and something I made for internal project. Idea being that you ideally only access maps last since they’re created in variables, and you may have operator functions you’d want to use in variables. For example, using rem-calc during a variable declaration.